algorithms/src/Kokkos_Sort.hpp

```diff
@@ -596,10 +578,21 @@ std::enable_if_t<Kokkos::is_execution_space<ExecutionSpace>::value> sort(
   bin_sort.sort(exec, view);
 }
 
+#if defined(KOKKOS_ENABLE_SERIAL)
+template <class DataType, class... Properties>
+void sort(const Serial& space,
+          const Kokkos::View<DataType, Properties...>& view) {
+  space.fence("Kokkos::sort: Fence before sorting on the host");
```

Comment:
Why would you need this fence (which is a no-op anyway)?

Reply:
It was originally in `try_std_sort()`.

Reply:
If we pass an execution space, we really shouldn't have this fence here.

Reply:
why should not fence the space instance here?

Reply:
oh wait this is before calling sort, so I agree

Reply:
Also, it's the overload taking an execution space instance.

Reply:
Removed.